Kajabi/sage-archived
# Button styles

## Description
- Updates buttons, switches, and radio styles with revised styling for:
  - focus
  - hover
  - active
  - disabled
- Converts border styles from `px` to `rem`
- Corrects body font style to match spec
- Scaffolding for future link button and icon button styles
- Lowercase hex codes (will be in future stylelint config)

![button-spec](https://user-images.githubusercontent.com/816579/74980625-aa6b5b00-53e5-11ea-80e2-e142652ce51e.png)
![controls](https://user-images.githubusercontent.com/816579/75081745-31dbcb80-54c5-11ea-9f2d-b9dd4e1e3c2d.png)


## Test notes
### Steps for testing
1. Start Sage with `rails s`
2. Navigate to "Elements" in the navigation
3. Verify that buttons, switches, and radio inputs match the [Figma spec](https://www.figma.com/file/xlUFoRTjsFePavVT85SmLL/Sage-Design-System-v1.0?node-id=1%3A2) for `:hover`, `:focus`, `:active`, and `:disabled`
4. Note that error states for switches and radio inputs are not currently implemented.

## Related issues
- Closes #31
